crates/types/src/services/p2p/peer_reputation.rs

```diff
@@ -0,0 +1,56 @@
+/// PeerScore type used for Peer Reputation
+pub type PeerScore = f64;
```

Comment:
It would be nice to have its own type for the application score and maybe `i32` or `i64` would be enough for us=)

The `PeerScore` is good naming for the application score and gossiping score. But if we make the gossiping score independent, then we need only `ApplicationScore`

Reply:
good point, `AppScore` is clearer name. I left `f64` type since gossipsub uses the same, and we can decay the score by ```current_score * 0.9``` as we discussed